### PR TITLE
Adds Maintenance page code

### PR DESCRIPTION
**Pull request by crwallace**
screen shot 2016-06-29 at 3 55 50 pm

### CODE REVIEW

**Comment by joduinn** [comment]
Possible tweak to change "You may call" to "In the meanwhile, you can still call ..."  ?

Only a nit, so feel free to ignore. And yes, I like it!

**Comment by staceylanger** [comment]
Agree with John's changes. I would simplify by removing "Bear with us".

**Comment by crwallace** [comment]
Copy edits below:
screen shot 2016-06-29 at 5 22 15 pm

File: public/index.html [hunk 1]
```diff
@@ -139,7 +141,34 @@ <h2 class="sr-only">Apply for Health Care</h2>
...
+  </div>
+
+<div class="main home" role="main">
+
+  <div class="section main-menu">
+    <div class="row">
+      <div class="small-12 columns">
+        <div style="padding: 2em 0;">
+        <h3>We're sorry. The healthcare application is currently down while we fix a few things. We will be up and running as soon as we can.</h3>
+        <h4>In the meantime, you can still call 1-877-222 VETS(8387) press 2 to complete this application over the phone.</h4>
```
Comment (akainic) [inline public/index.html]: Something looks odd with the telephone number. Maybe we need a `-` between `222` and `VETS` and a space between `VETS` and `(8387)`. Also it's `and press 2` afterwards, missing an `and`.

File: public/index.html [hunk 2]
```diff
@@ -139,7 +141,34 @@ <h2 class="sr-only">Apply for Health Care</h2>
     </div>
   </div>
 </div>
+<!-- HCA Application End -->
 
+<!-- Maintenance Page Start -->
+  <!--<div class="splash">
```
Comment (akainic) [inline public/index.html]: We're keeping this commented out so we can uncomment it and deploy if anything goes wrong?

File: public/index.html [hunk 3]
```diff
@@ -139,7 +141,34 @@ <h2 class="sr-only">Apply for Health Care</h2>
...
+    </div>
+  </div>
+
+<div class="main home" role="main">
+
+  <div class="section main-menu">
+    <div class="row">
+      <div class="small-12 columns">
+        <div style="padding: 2em 0;">
+        <h3>We're sorry. The healthcare application is currently down while we fix a few things. We will be up and running as soon as we can.</h3>
```
Comment (akainic) [inline public/index.html]: Maybe `We will be back up as soon as we can` instead of `up and running`? More clear?

File: public/index.html [hunk 4]
```diff
@@ -139,7 +141,34 @@ <h2 class="sr-only">Apply for Health Care</h2>
...
   </div>
 </div>
+<!-- HCA Application End -->
 
+<!-- Maintenance Page Start -->
+  <!--<div class="splash">
+    <div class="row">
+      <div class="small-12 columns">
+        <div class="pitch">
+          <h2 class="tagline"><span>Wednesday, June 29th, 2016 at 3:38 PM EST</span></h2>
```
Comment (akainic) [inline public/index.html]: Can we enter the current time somehow instead of hard-coding this?

File: public/index.html (hunk 2 above)
Comment (crwallace) [inline public/index.html]: Correct!

File: public/index.html (hunk 3 above)
Comment (crwallace) [inline public/index.html]: @maryannbrody weigh in if you have feelings about this.

File: public/index.html (hunk 4 above)
Comment (crwallace) [inline public/index.html]: We would need to capture the time of the deploy. We are trying to give the user an idea of how long that message has been up. Do we have a place where that timestamp is getting stored?

Comment (crwallace) [inline public/index.html]: @maryannbrody -- Do we think it is crucial for the user to know the time? I am thinking about removing it from the header. I started looking through other examples of system maintenance messages and very few have a timestamp.

File: public/index.html (hunk 3 above)
Comment (maryannbrody) [inline public/index.html]: @crwallace - no strong preference. @akainic's suggestion works for me.

**Comment by crwallace** [comment]
This closes: 
- 435: Create a redirect page if app is down

**Comment by akainic** [comment]
Looks great! Updating branch and then will merge once tests pass.

**Pull request merged**